renv.lock

```diff
@@ -1032,11 +1037,11 @@
       "Version": "0.0.6",
       "Source": "GitHub",
       "RemoteType": "github",
+      "RemoteHost": "api.github.com",
       "RemoteUsername": "ccao-data",
       "RemoteRepo": "lightsnip",
-      "RemoteRef": "master",
-      "RemoteSha": "beeb858b2c9a4022d784b4a672941d5801ae87c1",
-      "RemoteHost": "api.github.com",
+      "RemoteRef": "jeancochrane/save-record-evals",
```

Comment:
Pinning lightsnip to [this branch](https://github.com/ccao-data/lightsnip/pull/10) so that we can save `record_evals` for models during comp runs. I'll plan to merge https://github.com/ccao-data/lightsnip/pull/10 first and set this back to `master` before merging.